ProcessLib/HeatTransportBHE/HeatTransportBHEProcess.cpp

```diff
@@ -9,8 +9,8 @@
 
 #include "HeatTransportBHEProcess.h"
 
-#include <iostream>
 #include <cassert>
+#include <iostream>
```

Comment:
I don't think the iostream include is needed.

Reply:
removed.